directive/data.js

```diff
@@ -1,9 +1,11 @@
 import { directive } from "../core.js";
 import { effect } from "../signal.js";
 
+const lowerCaseFirst = str => `${str.charAt(0).toLowerCase()}${str.slice(1)}`;
+
 directive['data'] = (el, evaluate, state) => {
   return effect(() => {
     let value = evaluate(state)
-    for (let key in value) el.dataset[key] = value[key];
+    for (let key in value) el.dataset[lowerCaseFirst(key)] = value[key];
```

Comment:
Does that cause a problem? I sort of relied on default means. Do you have a test for that?

Reply:
@dy No, the problem is not here, the problem is that this code is not at all in **dist/sprae.js**!

Reply:
@vanodevium I mentioned in readme that it's optional directive, you have to plug it in via `import sprae/directive/data`. But it gives me a thought that for standalone bundle it would make sense to include it by default. I am not convinced yet about `:html` directive though, it is still experimental and I don't have convincing use-cases for it.